### ISSUE TITLE
Configurable slide dividers

### ISSUE THREAD

**Issue by sfairchild** [jedcn/reveal-ck#88]
I just came up with an issue while making a presentation where the vertical dividers `***` were messing up my syntax highlighting (it thinks I am trying to bold the section). It would be a nice feature if we could define the section separators inside the config.yml. We can keep the current defaults, but add the flexibility. 

For example

```
slide_seperator: ---
vertical_slide_seperator: --
```

If this is something that you want I can implement it and give a pull request.

**Comment by jedcn** [comment]
Hey, there, @sfairchild! Thanks for creating an issue.

I'd never thought about it this-- and if you think you (or people you know) would benefit from it then I don't see why we could have:

1. A default, and
2. The ability to configure reasonable alternatives

Just curious-- what editor are you using and how'd you learn about this project?

If you'd like to put together a PR you can-- but as a heads up-- it takes me awhile to get into this code and I wrote it :smile:.

I'd likely get some free time in the next week or so and can take a look.

Best,

Jed

**Comment by jedcn** [comment]
Was thinking about this a bit and recalled the reason I selected `***` for vertical dividers is because they are a valid equivalent of `---`: https://cl.ly/3F0E0o0i3z2O

If that's the case then your editor's syntax highlighting is implemented incorrectly, right?

**Comment by sfairchild** [comment]
Thanks for the response. 

I am using vim to create the presentation and just using the stock markdown syntax highlighting, not sure why but it thought 
```
***
Slide Contents
***
```
was supposed to be bolded (`*bold*` is how you bold md). 

![bad_md](https://user-images.githubusercontent.com/1839202/30706127-6b572750-9ec5-11e7-974c-f552579b5e01.png)


I switched to the 'plasticboy/vim-markdown' plugin and it was highlighting correctly.

I just switched to this reveal-ck after using a yeoman reveal.js generator after finding reveal-ck by accident. The yeoman generator uses `---` for regular slides and `--` for vertical slides. That is where I came up with the idea for different slide separators.

Since I came across this issue I switched to haml for the extra control.

btw, I really like what you have created with this project, very simple and fast to use. Thanks you for it.
